sdk/spring/spring-cloud-azure-starter-monitor/src/main/resources/META-INF/native-image/native-image.properties

```diff
@@ -1,7 +1,5 @@
 Args=\
-  --initialize-at-build-time=ch.qos.logback.core \
-  --initialize-at-build-time=ch.qos.logback.classic \
-  --initialize-at-build-time=io.opentelemetry.instrumentation.logback.appender.v1_0.internal.LoggingEventMapper \
-  --initialize-at-build-time=io.opentelemetry.instrumentation.api.internal.cache.concurrentlinkedhashmap.ConcurrentLinkedHashMap \
-  --initialize-at-build-time=io.opentelemetry.api.DefaultOpenTelemetry \
-  --initialize-at-build-time=io.opentelemetry.context.propagation.DefaultContextPropagators
+    --initialize-at-build-time=ch.qos.logback \
+    --initialize-at-build-time=org.slf4j.helpers \
+    --initialize-at-build-time=com.fasterxml.jackson.core.Base64Variant \
```

Comment:
This line and the following one seem related to fasterxml provided by an OTel transient dependency